src/apis/product.apis.ts

```
@@ -49,15 +57,40 @@ export const updateQuantity = (body: {
 export const getOrders = (
   token: string,
   page: number = 1,
-  status?: "PENDING" | "IN_TRANSIT" | "COMPLETED" | "MERCHANT_ACCEPTED"
+  status?: | "PENDING"
+  | "IN_TRANSIT"
+  | "DELIVERED"
+  | "MERCHANT_ACCEPTED"
+  | "MERCHANT_REJECTED"
+) => {
+  let queryParams = "";
+  if (!!status) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)